packages/react/src/components/Accordion/Accordion.mdx

```diff
@@ -94,7 +94,9 @@ through to the underlying accordion header.
 
 The `className` prop passed into `AccordionItem` will be forwarded along to the
 underlying accordion header. This is useful for specifying a custom class name
-for layout.
+for layout. As a reminder, the callback signature will look like
+`{ isOpen: boolean, event: Event }` rather than the usual
+`event: Event, state: { isOpen: boolean }` until the next major version.
```

Comment:
```suggestion
`({ isOpen: boolean, event: Event })` rather than the usual
`(event: Event, state: { isOpen: boolean })` until the next major version.
```

I guess I should've wrapped these with parentheses earlier in my comment